index.html

```diff
@@ -34,6 +34,72 @@
             </div>
         </div>
     </div>
+
+    <div class="container-fluid">
+        <nav class="navbar navbar-expand-sm navbar-dark" style="background-color: #2196F3;">
+            <button style="background-color: rgba(0, 132, 255, 0.788);" class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapsibleNavId" aria-controls="collapsibleNavId" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="collapsibleNavId">
+                <ul class="navbar-nav ml-auto">
+                    <li class="nav-item active">
+                        <a class="nav-link" href="#">Home page</a>
+                    </li>
+                    <li class="nav-item active" style="margin-left: 50px;">
+                        <a class="nav-link" href="#">About us</a>
+                    </li>
+                    <li class="nav-item dropdown active" style="margin-left: 50px;">
+                        <a class="nav-link dropdown-toggle" href="#" id="dropdownId" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Available furniture</a>
+                        <div class="dropdown-menu" aria-labelledby="dropdownId">
+                            <a class="dropdown-item" href="#">Office chairs</a>
+                            <a class="dropdown-item" href="#">Dinner chairs</a>
+                            <a class="dropdown-item" href="#">Rocking chairs</a>
+                            <a class="dropdown-item" href="#">Glider chairs</a>
+                        </div>
+                    </li>
+                    <li class="nav-item active" style="margin-left: 50px;">
+                        <a class="nav-link" href="#">News</a>
+                    </li>
+                    <li class="nav-item active" style="margin-left: 50px;">
+                        <a class="nav-link" href="#">Contact us</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </div>
+
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-3">
+                <nav class="navbar navbar-expand-sm navbar-dark flex-lg-column" style="background-color: #2196F3 ; align-items: unset;">
+                    <p style="font-size: 20px; color: white;">Avaliable brands</p>
+                    <button style="background-color: rgba(0, 132, 255, 0.788);" class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapsibleNavId" aria-controls="collapsibleNavId" aria-expanded="false" aria-label="Toggle navigation">
+                        <span class="navbar-toggler-icon"></span> 
+                    </button>
+                    <div class="collapse navbar-collapse navbar-dark" id="collapsibleNavId">
+                        <ul class="navbar-nav mt-2 flex-lg-column">
```

Comment:
remove mt-2. We do not adjust the child position for the parent

Reply:
done